src/default-namespace.ts

```diff
@@ -27,6 +27,9 @@ const firebaseAdmin = new FirebaseNamespace();
 //
 //   import * as firebaseAdmin from 'firebase-admin';
 //   which becomes: var firebaseAdmin = require('firebase-admin');
-(firebaseAdmin as any).default = firebaseAdmin;
+// (firebaseAdmin as any).default = firebaseAdmin;
 
-export = firebaseAdmin;
+// export = firebaseAdmin;
+
+export * from './firebase-namespace';
+export * from './firebase-app';
```

Comment:
I feel like this entire file could be removed, and we could only have the existing runtime check in the `index.ts` along with the barrel exports. 

As-per our previous conversation only named imports are supported. Firstly, we want to have `./firebase-app` to be included in this so that users can import `FirebaseApp`. Secondly, we also want aspects from `./firebase-namespaces` such as `initializeApp`. 

Currently, I added a global constant object that is used for the global namespace, which is used as a substitute of the `INTERNAL` field before in the firebase-namespace.

Reply:
Yeah, this file should eventually disappear. `index.ts` will be the only entry point to the package.